src/algorithms/reco/FarForwardNeutronReconstruction.cc

```diff
@@ -6,14 +6,14 @@
 #include <edm4hep/Vector3f.h>
 #include <math.h>
 #include <gsl/pointers>
-#include <vector>
 #include <stdexcept>
+#include <vector>
 
 #include "FarForwardNeutronReconstruction.h"
 
 /**
- Creates a "neutron candidate" Reconstructed Particle consisting of all clusters in a
- given ClusterCollection.  Its energy is the sum of the energies of the constituent clusters
+ Creates a "neutron candidate" Reconstructed Particle consisting of all clusters in the
+ ZDC (both Hcal and Ecal collections).  Its energy is the sum of the energies of the constituent clusters
```

Comment:
Remember, we want algorithms to be as generic as possible: while the ZDC will likely be the primary user of this algorithm for the time being, this could be used by other combinations of subsystems. So please make sure code and comments reflect this.

Not to mention, the ECal clusters are an optional input, so how about the following?
```
Creates a candidate neutron by summing the energy of all clusters from a specified hadronic calorimeter and (optionally) from a specified Electromagnetic calorimeter.
```

Reply:
Thanks, done